defaults/main.yml

```diff
@@ -49,6 +49,7 @@ ansistrano_git_branch: master
 ansistrano_git_repo_tree: ""
 ansistrano_git_identity_key_path: ""
 ansistrano_git_identity_key_remote_path: ""
+ansistrano_git_executable: ""
```

Comment:
I thinks it's better to not add an empty default as the default is to omit it (like we have for refspec)